src/app/Components/EthrDid/helpers.ts

```diff
@@ -0,0 +1,4 @@
+import { DIDDocument, PublicKey } from 'did-resolver'
+
+export const getOwnerFromDidDoc = (didDocument: DIDDocument) =>
```

Comment:
This is called _selector_ in Redux world :) You can just out this function at the bottom of the reducer. All consumers of that reducer should always use the selector. Then, when the state changes the consumer does not need updates

Reply:
It should get as parameter the `ethrdid` state to keeps it even more flexible.

Reply:
In the refactor of containers, this file got removed and the function is non-exportable inside of DashboardContainer.